### PR TITLE
Show time

### PR DESCRIPTION
**Pull request by anshul2401**
Unable to display time on detail screen. showing 'null'.

### CODE REVIEW

**Comment by harsimranmaan** [comment]
The code looks good to me. The null happens because the tasks are fetched from the local repo by default. Since those are already saved on the phone, the webclient does not come into picture. See https://github.com/bigpanther/shipanther/blob/main/lib/tasks_repository_local_storage/repository.dart#L30-L32 for the flow. If you temporarily change Line30 with line 32, you can see the time is shown properly. We need to come up with a better strategy around cache invalidation.

**Comment by harsimranmaan** [comment]
One easy to handle this would be to add a refresh icon to the page. When the page is refreshed, we explicitly refresh the content.

File: lib/blocs/models/task.dart [hunk 1]
```diff
@@ -13,8 +13,9 @@ class Task {
   final String containerName;
   final String from;
   final String to;
+  final DateTime pickingTime;
 
-  Task(this.containerName,
+  Task(this.containerName, this.pickingTime,
```
Comment (harsimranmaan) [inline lib/blocs/models/task.dart:18]: This should be renamed to pickup time

**Comment by anshul2401** [comment]
> The code looks good to me. The null happens because the tasks are fetched from the local repo by default. Since those are already saved on the phone, the webclient does not come into picture. See https://github.com/bigpanther/shipanther/blob/main/lib/tasks_repository_local_storage/repository.dart#L30-L32 for the flow. If you temporarily change Line30 with line 32, you can see the time is shown properly. We need to come up with a better strategy around cache invalidation.
> 
> port

**Comment by anshul2401** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by nicoletamaan — APPROVED** [review]

**Comment by harsimranmaan** [comment]
Outdated

**Pull request closed without merging**